iphone/Maps/Core/DeepLink/DeepLinkHandler.swift

```diff
@@ -105,6 +105,14 @@ class DeepLinkURL {
     isLaunchedByDeeplink = false
     deeplinkURL = nil
   }
+  
+  func getBackUrl() -> String {
+    if deeplinkURL != nil {
```

Comment:
Maybe 
`guard let urlString = deeplinkURL?.url.absoluteString else { return "" }
`?
Force unwrap on the next line isn't safe